lib/taskana-core-test/src/test/java/acceptance/classification/query/QueryClassificationAccTest.java

```diff
@@ -0,0 +1,984 @@
+            .asSummary();
+    assertThat(classificationSummaryList)
+        .containsExactlyInAnyOrder(
+            externalClassificationSummary,
+            manualClassificationSummary,
+            masterClassificationExternal,
+            masterClassificationManual);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassifications_When_QueryingForParentId() throws Exception {
+    String testIdentifier = UUID.randomUUID().toString();
+    ClassificationSummary parentClassificationSummary =
+        defaultTestClassification()
+            .customAttribute(CUSTOM_1, testIdentifier)
+            .buildAndStoreAsSummary(classificationService);
+    ClassificationSummary childClassificationSummary =
+        defaultTestClassification()
+            .customAttribute(CUSTOM_1, testIdentifier)
+            .parentId(parentClassificationSummary.getId())
+            .buildAndStoreAsSummary(classificationService);
+
+    List<ClassificationSummary> classificationSummaryList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .parentIdIn(parentClassificationSummary.getId())
+            .list();
+
+    ClassificationSummary masterChildClassification =
+        classificationService
+            .getClassification(childClassificationSummary.getKey(), "")
+            .asSummary();
+    assertThat(classificationSummaryList)
+        .containsExactlyInAnyOrder(childClassificationSummary, masterChildClassification);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassifications_When_QueryingForParentKey() throws Exception {
+    String testIdentifier = UUID.randomUUID().toString();
+    ClassificationSummary parentClassificationSummary =
+        defaultTestClassification()
+            .customAttribute(CUSTOM_1, testIdentifier)
+            .buildAndStoreAsSummary(classificationService);
+    ClassificationSummary childClassificationSummary =
+        defaultTestClassification()
+            .customAttribute(CUSTOM_1, testIdentifier)
+            .parentKey(parentClassificationSummary.getKey())
+            .buildAndStoreAsSummary(classificationService);
+
+    List<ClassificationSummary> classificationSummaryList =
+        classificationService
+            .createClassificationQuery()
+            .parentKeyIn(parentClassificationSummary.getKey())
+            .list();
+
+    ClassificationSummary masterChildClassification =
+        classificationService
+            .getClassification(childClassificationSummary.getKey(), "")
+            .asSummary();
+    assertThat(classificationSummaryList)
+        .containsExactlyInAnyOrder(childClassificationSummary, masterChildClassification);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @TestFactory
+  Stream<DynamicTest> should_FindClassifications_When_QueryingForCustomLikeFields() {
```

Comment:
Let's move the test right below the  `should_FindClassifications_When_QueryingForCustomInFields()` That way, the order makes more sense.

Reply:
Did it, however changed the name to should_FindClassifications_When_QueryingForCustomAttributeLike() since the name of the other test was should_FindClassifications_When_QueryingForCustomAttributeIn().